docs/getting-started/sfx-vs-nuxt.md

```diff
@@ -0,0 +1,57 @@
+# SFX vs. Nuxt
+
+Our aim is to be an e-commerce framework. Our goal is modularity and speed. Through our long standing experience with Nuxt, we came to understand, that complex projects need modular architecture, full control of build packages and ability to have multiple build configurations from one repository (e.g. different frontends in different countries).
+
+## What is similar?
+
+Storefront X is the same as Nuxt, both are server-side rendered applications. Our framework also uses folder enforced structure, so you can work with `pages` directory as you would work in Nuxt. We also use Vue as our component application framework, but unlike Nuxt, Storefront X can work with other frameworks like React or SolidJS.
+
+## Where we see our advantage?
+
+We've taken the folder abstraction from Nuxt and taken it a step further. Because a complex e-commerce solution needs to work in a multiverse of frontend versions, we added another level of abstraction in the form of modules. For this abstraction to work with small build bundles, we don't import things directly from modules, but from the abstraction folder created at build time. That way we can override, extend and change behaviour with each new module, without code refactoring in existing modules.
+
+## SFX as a generated framework
+
+Also, while Nuxt is a complex framework, it is not a modular one. It is a monolith, which is hard to extend and change. Storefront X is a modular framework, which allows you to change, extend and override any part of the framework, because in depth, all custom created concepts are generated into the build folder, as it was explain in the section above. This allows Storefront X to provide application authors with a ton of functionality, while never limiting or slowing down the application development.
```

Comment:
First half good. Sexond not so much. Give an emphasis to the fact that third party modules can easily add any functionality to storefront x (core sfx is simple and everything added via modules). Also drop mention of generation.